src/app/controllers/CartController.ts

```diff
@@ -29,6 +29,14 @@ export interface CartCoupon {
   _id: string;
   coupon: Coupon;
 }
+export interface ParticipatingCustomer {
```

Comment:
Shouldn't this be Pick<Customer, _id, name etc btw? Why create a new type if it all comes from customer